pom.xml

```diff
@@ -135,7 +135,7 @@
         <deploy.skip>true</deploy.skip>
 
         <!-- BOM-referenced versions -->
-        <jakartaee.api.version>9.1.0</jakartaee.api.version>
+        <jakartaee.api.version>10.0.0-SNAPSHOT</jakartaee.api.version>
```

Comment:
If we're adding SNAPSHOT to the pom, do we need to temporarily add the Jakarta staging repo? This won't build unless we add it

Reply:
this change is temporal just to test the samples, and as you mention it is needed to build the jakarta staging repo. When the last version of the jakartaee.api is uploaded with last changes we just need to upgrade to that version. I will take care to change this to an available version of the jakartaee.api before to merge to the payara6

Reply:
I will add it in my PR right after this one. Whole this branch is temporal until we will have Concurrent RI 3.0 implemented and published.